common/source/docs/common-datagnss-gem1305-rtk.rst

```diff
@@ -0,0 +1,206 @@
+.. _common-datagnss-gem1305-rtk:
+
+==================================
+GEM1305 RTK receiver with antenna
+==================================
+
+This page mainly includes some instructions for using the GEM1305 and NANO RTK Receiver products with Ardupilot.
+
+GEM1305 is a RTK receiver with antenna designed and manufactured by DATAGNSS. 
+
+GEM1305 is based on the new generation Allystar dual-core GNSS SoC. It supports RTK functionality with a maximum data update rate of 5Hz. 
+
+.. image:: ../../../images/gem1305/datagnss-gem1305-02.png
+	:target: ../_images/gem1305/datagnss-gem1305-02.png
+
+.. image:: ../../../images/gem1305/datagnss-gem1305-03.png
+	:target: ../_images/gem1305/datagnss-gem1305-03.png
+
+
+GEM1305/NANO RTK Receiver doesn't support Moving Base mode.
+
+Where to Buy
+============
+
+- `DATAGNSS website <https://www.datagnss.com/>`_
+
+Key Features
+============
+
+- Full constellation, multi-frequency GNSS satellite receiver
+- Support RTK
+- Standard UART serial interface
+- Lightweight only 50g or 26g(NANO RTK receiver)
+- High performance antenna
+
+
+Frequencies
+===========
+   -  GPS/QZSS: L1 C/A, L5C
+   -  GLONASS: L1OF
+   -  BEIDOU: B1I, B2a
+   -  GALILEO: E1, E5a
+
+GNSS
+====
+   - Allystar CYNOSURE IV GNSS SOC
+   - Dual core
+   - 3D accuracy: **1.5m** CEP
+   - RTK accuracy: **2cm** +1PPM(H), 3cm+1PPM(V)
+
+Interface
+=========
+   - UART, 230400bps
+   - SMA for antenna
+   - Output rate 5Hz
+   - Main power supply,4.7~5.2V
+
+Protocol
+========
+   - NMEA output
+   - RTCM v3 input/output
+
+Environment
+===========
+   - Operating temp. -20~85℃
+
+Dimession and weight
+====================
+   - 55*55*12mm
+   - 50g (GEM1305) 26g (NANO with Helix)
+
+
+Pin definition
+==============
+
+The board is connected to the autopilot via UART interface.
+
+.. image:: ../../../images/gem1305/GEM1305-connector.png
+	:target: ../_images/gem1305/GEM1305-connector.png
+
+The 1.25mm pitch 6P connector (from Left PIN1 to PIN6) :
+
+   -  1: GND
+   -  2: NC
+   -  3: PPS
+   -  4: Rx
+   -  5: Tx
+   -  6: 5V
+
+Please note that the board only includes RTK and doesn't inlcude compass.
+
+ArduPilot Setup
+===============
+
+Assuming GEM1305 is connected to AP's SERIAL4(2nd GPS), it should work with the default configuration.
+
+- :ref:`SERIAL3_PROTOCOL <SERIAL4_PROTOCOL>` = 5
+- :ref:`SERIAL4_BAUD <SERIAL4_BAUD>` = 230
```

Comment:
dont think this is needed with autodetect protocol...it will autobaud to this

Reply:
OK,let me update it.